advocate_europe/assets/scss/components/_blocks.scss

```diff
@@ -125,9 +125,26 @@
     }
 
     .blockblog-image {
+        background-size: cover;
+        background-repeat: no-repeat;
+        display: block;
+        position: relative;
+        box-sizing: content-box;
```

Comment:
I don't understand why this is needed (Doesn't mean that it is not needed).

Reply:
You're right, isn't needed.